server/base/src/main/java/org/apache/accumulo/server/problems/ProblemReports.java

```diff
@@ -68,7 +68,7 @@
    * is reporting lots of problems, but problem reports can not be processed
    */
   private ExecutorService reportExecutor = ThreadPools.createThreadPool(0, 1, 60, TimeUnit.SECONDS,
-      "acu-problem-reporter", new LinkedBlockingQueue<>(500), OptionalInt.empty(), false);
```

Comment:
The removal of this boolean makes the changeset a lot harder to review. I'd suggest avoiding that and adding it in a quick second commit after this PR is merged, and just ignoring the parameter in the method for now, so the changeset is smaller and easier to review, but it's probably easier to just leave it alone, since it's already done.

Reply:
I added the boolean back to the methods (or overloaded methods) and reverted all of the calls to reduce the noise in the PR in commit 89a814b